docs/pages/guides/spend-permissions/quick-start.mdx

```diff
@@ -0,0 +1,702 @@
+brand new [Coinbase Smart Wallet](/why.mdx). The address of a smart contract wallet is deterministic, and depends on the bytecode of the implementation
+contract combined with a salt, which in our case will be an array consisting of the initial owner(s) of the smart contract wallet. See the 
+[smart wallet repository](https://github.com/coinbase/smart-wallet) for more details. 
+
+We'll generate the deterministic address of this Smart Wallet
+so we can store it as a public environment variable.
+
+Paste the following code into a top-level file called `logSpenderSmartWalletAddress.ts`
+
+```ts [logSpenderSmartWalletAddress.tsx]
+import { createPublicClient, Hex, http } from "viem";
+import { baseSepolia } from "viem/chains";
+import { privateKeyToAccount } from "viem/accounts";
+import {
+  toCoinbaseSmartAccount,
+} from "viem/account-abstraction";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+export async function logSpenderSmartContractWalletAddress() {
+  const client = createPublicClient({
+    chain: baseSepolia,
+    transport: http(),
+  });
+
+  const spenderAccountOwner = privateKeyToAccount(
+    process.env.NEXT_PUBLIC_SUBSCRIPTION_PRIVATE_KEY! as Hex
+  );
+  console.log("spenderAccountOwner", spenderAccountOwner.address);
+
+  const spenderAccount = await toCoinbaseSmartAccount({
+    client,
+    owners: [spenderAccountOwner],
+  });
+console.log("Spender Smart Wallet Address:", spenderAccount.address);
+}
+
+async function main() {
+  await logSpenderSmartContractWalletAddress();
+}
+
+if (require.main === module) {
+  main().catch((error) => {
+    console.error(error);
+    process.exit(1);
+  });
+}
+```
+
+Run this script to log the counterfactual address of your new smart contract wallet and assign this address to the 
+`NEXT_PUBLIC_SUBSCRIPTION_SPENDER` variable in your .env.
+
+<Callout type="info">
+You may need to install ts-node or other of the following dependencies: `npm install -g ts-node typescript @types/node dotenv`  
+</Callout>
+
+<Callout type="info">
+You may need to temporarily set the value of `"module"` in your `tsconfig.json` to `"commonjs"` to run this script.
+</Callout>
+
+
+
+### Set up remaining environment variables
+Next, make sure you have a Coinbase Developer Platform API key, which you can get [here](https://portal.cdp.coinbase.com/). 
+Assign this key to `NEXT_PUBLIC_CDP_API_KEY` in your `.env`.
+
+You'll need one more environment variable, which is `BASE_SEPOLIA_PAYMASTER_URL`.  
+This one's easy if you already have your CDP API key:  
+  `"https://api.developer.coinbase.com/rpc/v1/base-sepolia/{YOUR_CDP_API_KEY}"`
+
+
+### Create a spender client
+
+Our client is what our app will use to communicate with the blockchain. In this example, our client is a Coinbase Smart Wallet, 
+and we'll use a [paymaster](/guides/paymasters.mdx) to sponsor our transactions so we don't have to worry
+about having gas money.
```

Comment:
nit: somewhat misleading as we will have to pay CDP back 😅 maybe "so we don't have to worry about having ETH in the spender account"